CONTRIBUTING.md

```diff
@@ -4,3 +4,8 @@
 3. create new unittests as necessary
 4. update the README if applicable
 5. update the setup script if applicable
+
+# Setting up a development environment
+1. After checking out this repository, create a python3 environment (in ./venv) with `python3 venv venv`
+2. Install all pkpass prerequisites into that virtual environment by runnint `./venv/bin/pip install -r requirements.txt`
```

Comment:
typo `runnint` should be `running`

prior to lines 10 and 11; I would suggest the user doing a source i.e.
`source venv/bin/activate`

And then lines 10 and 11 can instead be
```suggestion
2. Install all pkpass prerequisites into that virtual environment by running `pip install -r requirements.txt`
```
and 
```
3. You should be able to run the latest checked out pkpass with `python pkpass.py`
```